tests/impl.cpp

```diff
@@ -194,6 +195,37 @@ static inline float bankersRounding(float val)
     return ret;
 }
 
+static inline double bankersRounding(double val)
+{
+    if (val < 0)
+        return -bankersRounding(-val);
+
+    double ret;
+    double roundDown = floor(val);  // Round down value
+    double roundUp = ceil(val);     // Round up value
+    double diffDown = val - roundDown;
+    double diffUp = roundUp - val;
+
+    if (diffDown <
```

Comment:
Rewrite as following
```c
// If it's closer to the round down value, then use it
if (diffDown < diffUp) {
```

Reply:
I wonder where should I put the comment in this case:
```c
    } else if (diffDown >
               diffUp) {  // If it's closer to the round up value, then use it
        ret = roundUp;
```

Reply:
Let's make the condition expression as clear as possible.

Reply:
Done.